File: demo05.pl
#!/usr/bin/perl -w
$line = "complex regex statement";
print "line = $line\n";
print "$line = complex lower case\n" if $line =~ /complex/;
print "$line = complex ignore case\n" if $line =~ /COMPLEX/i;
$line =~ s/regex/reGX/i;
print "$line = regex substituted for regX\n";
$line =~ s/regx/reX/i;
print "$line = regX substituted for reX\n";
$line =~ s/e/5/gi;
print "$line = replace all e for 5\n";
print "line = $line\n";

File: tests/demo05.py
#!/usr/bin/python2.7 -u
import re
line = "complex regex statement"
print "line = " + line
if re.search(r'complex', line) : print line + " = complex lower case"
if re.search(r'(?i)COMPLEX', line) : print line + " = complex ignore case"
line = re.sub(r'(?i)regex', 'reGX', line)
print line + " = regex substituted for regX"
line = re.sub(r'(?i)regx', 'reX', line)
print line + " = regX substituted for reX"
line = re.sub(r'(?i)e', '5', line)
print line + " = replace all e for 5"
print "line = " + line
